File: Objetivo 1/Objetivo1_Exercicios_PHP/model/Produto.php
<?php

/**
 * Description of Produto
 *
 * @author vagner
 */
class Produto {
    private $nome;
    private $descricacao;
    private $valor;
    private $quantidade;
    
    public function __construct() {
        
    }
    
    public function getNome() {
        return $this->nome;
    }

    public function getDescricacao() {
        return $this->descricacao;
    }

    public function getValor() {
        return $this->valor;
    }

    public function getQuantidade() {
        return $this->quantidade;
    }

    public function setNome($nome) {
        $this->nome = $nome;
    }

    public function setDescricacao($descricacao) {
        $this->descricacao = $descricacao;
    }

    public function setValor($valor) {
        $this->valor = $valor;
    }

    public function setQuantidade($quantidade) {
        $this->quantidade = $quantidade;
    }

    public function __toString() {
        return "Produto{nome=$this->nome, descricacao=$this->descricacao, valor=$this->valor, quantidade=$this->quantidade]";
    }
}

File: Objetivo 1/Objetivo1_Exercicios_Java/src/model/Produto.java
package model;

/**
 *
 * @author vagner
 */
public class Produto {
    private String nome;
    private String descricacao;
    private double valor;
    private int quantidade;

    public Produto() {
    }

    public Produto(String nome, String descricacao, double valor, int quantidade) {
        this.nome = nome;
        this.descricacao = descricacao;
        this.valor = valor;
        this.quantidade = quantidade;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getDescricacao() {
        return descricacao;
    }

    public void setDescricacao(String descricacao) {
        this.descricacao = descricacao;
    }

    public double getValor() {
        return valor;
    }

    public void setValor(double valor) {
        this.valor = valor;
    }

    public int getQuantidade() {
        return quantidade;
    }

    public void setQuantidade(int quantidade) {
        this.quantidade = quantidade;
    }

    @Override
    public String toString() {
        return "Produto{" + "nome=" + nome + ", descricacao=" + descricacao + ", valor=" + valor + ", quantidade=" + quantidade + '}';
    }
    
}
